Gemfile.lock

```diff
@@ -359,10 +362,11 @@ DEPENDENCIES
   spring-watcher-listen (~> 2.0.0)
   timecop
   tty-prompt
+  validate_url (~> 1.0.8)
   webmock
 
 RUBY VERSION
    ruby 2.6.2p47
 
 BUNDLED WITH
-   2.0.2
+   2.1.4
```

Comment:
changing the bundler version doesn't seem part of this PR?

Reply:
sorry, my bad